conconartist/lyric-lava
# Prompt component

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Prompt component has functionality with displaying a new prompt from the Words API when the prompt button is clicked.  Basic CSS layout is complete with visible search bars and title.  Cypress testing is complete for this component.  

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js and Prompt.js in the components folder were edited, apiCalls.js in the src folder was updated with the fetch call for a random prompt, along with Prompt.css and App.css.  
App.spec.js in the cypress folder was updated with test blocks.


### How should this be tested?
> What steps need to be taken to confirm functionality? 

Go to the project directory and run `npm start` in your terminal to start the server and view the layout and functionality in your browser window when visiting the url http://localhost.3000.

Open the cypress folder in the project directory and enter `npx cypress open` to run cypress tests.


#### Checklist:
